src/test/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelperTest.java

```diff
@@ -0,0 +1,104 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Maps;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by liujoshua on 3/18/16.
+ */
+public class DynamoDataPipelineHelperTest {
```

Comment:
Can we add a test for createPipelineObjects() to make sure all the code is being exercised?